api/src/main/java/org/cardanofoundation/rosetta/config/CacheConfig.java

```diff
@@ -0,0 +1,17 @@
+package org.cardanofoundation.rosetta.config;
+
+import org.springframework.cache.annotation.EnableCaching;
+import org.springframework.cache.concurrent.ConcurrentMapCacheManager;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+import org.springframework.scheduling.annotation.EnableScheduling;
+
+@Configuration
+@EnableCaching
+@EnableScheduling
```

Comment:
Do we need here the `@EnableScheduling` annotation?

Reply:
Actually, you're right. We already have the @EnableScheduling annotation on the RosettaApiApplication class.